app/src/analytics/analytics-client.ts

```diff
@@ -0,0 +1,91 @@
+import crypto from 'crypto';
+
+interface AnalyticsAttributes {
+    [key: string]: any;
+}
+
+interface EventPayload {
+    eventName: string,
+    action: string,
+    actionSubject: string,
+    attributes?: AnalyticsAttributes
+}
+
+enum EnvType {
+    LOCAL = 'local',
+    DEV = 'dev',
+    STAGING = 'staging',
+    PROD = 'prod'
+}
+
+// eslint-disable-next-line max-len
+export const sendAnalytics = async (cloudId: string, eventPayload: EventPayload, accountId?: string, anonymousId?: string): Promise<void> => {
+    sendEvent(cloudId, eventPayload, accountId, anonymousId)
+        .then(() => console.log('Analytics event processed'))
+        .catch((e) => console.error({ e }, 'Failed to send analytics event'));
+};
+
+// eslint-disable-next-line max-len
+const sendEvent = async (cloudId: string, eventPayload: EventPayload, accountId?: string, anonymousId?: string): Promise<void> => {
+    const analyticsClient = await getAnalyticsClient();
+    const {
+        eventName, attributes, actionSubject, action
+    } = eventPayload;
+
+    if (!analyticsClient || !isProductionEnv()) {
+        console.warn('Analytics Node Client module not found or not prod. Ignoring the dependency.');
```

Comment:
/nit This warning makes sense in context but may not make sense on the console. Could we add what went wrong here? e.g. prefix "Sending analytics event failed. Analytics Node Client module not found or action not performed in prod evnvironment. Ignoring the dependency."

Reply:
fair